test/hex_web/api/router_test.exs

```diff
@@ -38,6 +39,20 @@ defmodule HexWeb.API.RouterTest do
     :ok
   end
 
+  @doc """
+  Creates and returns an auth key for a given user
+  """
+  @spec test_key_for(String.t) :: String.t
```

Comment:
Spec needs to be defined once as `@spec test_key_for(String.t | User.t) :: String.t`.